### ISSUE TITLE
Uncertain future

### ISSUE THREAD

**Issue by Meziu** [rust3ds/test-runner#18]
After https://github.com/rust3ds/ctru-rs/issues/171, the future of `test-runner` seems dim. Download links for Citra are unreachable and the emulator is officially dead.

@ian-h-chamberlain With the current state of things, and the fact we already planned on merging this repo back into `ctru-rs`, what are the remaining components that we can salvage from `test-runner`? Glancing over the project, it seems that anything more than just building the tests needs to be taken down.

**Comment by ian-h-chamberlain** [comment]
I think the plan I mentioned in https://github.com/rust3ds/test-runner/issues/5#issuecomment-1879908052 probably still applies, whether we decide to continue with Citra or not. Basically
- Move `test-runner` crate into `ctru-rs`, since the GDB / socket / console runners will probably still be useful, even on real hardware 
- Keep this repo as a GitHub Actions repo, with at least the `setup` action still for devkitPro + rust toolchain containers
- Either keep the `run-tests` action as-is, change it to use a different emulator, or delete it, depending on the outcome of discussion on https://github.com/rust3ds/ctru-rs/issues/171

Does that make sense?

**Comment by Meziu** [comment]
Let’s try to hook the emulated tests to a different source of Citra, and everything that isn’t a github action can be merged into `ctru-rs`.

**Comment by AzureMarker** [comment]
I think continuing with a different source of Citra is a good option for now, and it will probably be just fine for a long time. Investigating other options can happen in parallel.
